ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/transactions/WireTransaction.kt

```diff
@@ -0,0 +1,19 @@
+package net.corda.v5.ledger.common.transactions
+
+import net.corda.v5.application.serialization.SerializationService
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.DoNotImplement
+import net.corda.v5.crypto.SecureHash
+
+@DoNotImplement
+@CordaSerializable
+interface WireTransaction {
+    val id: SecureHash
+    val privacySalt: PrivacySalt
+    val componentGroupLists: List<List<ByteArray>>
+
+    fun getComponentGroupList(componentGroupId: Int): List<ByteArray>
+
+    fun getWrappedLedgerTransactionClassName(): String
```

Comment:
^ Is this needed for deserialisation later?

Reply:
We'll need something to know what is wrapped inside the wireTx which deserializes componentgroup 0,0 and gets the ledgerModel which will be the classname.